### ISSUE TITLE
When you hit Add Entity on an Enumerator question, auto-focus on the new input field

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/civiform#1483]
When I'm answering an Enumerator question, I will hit Add Entity to add a new input field and add a new nickname for a household member, job, car, etc. But when I hit Add Entity, the new input field that gets created does not get auto-focused, which is what i would expect.

This is a little annoying for users using a mouse, since they have to find and click into the new field, but it's problematic for users on screen readers, since they have to navigate backward through the form to find the new input area. If focus automatically moved to the new input box, no work would be needed on their part.

![image](https://user-images.githubusercontent.com/77128518/123303949-36f2c580-d4d3-11eb-996c-c433a6c517be.png)

So, once we notice the Add Entity button is pressed, and once we've constructed the new input field, let's move HTML's focus to it.

## Original tester's report

> Steps to reproduce:1. Navigate to URL: https://staging.seattle.civiform.com
> 2. Hit "Continue" on the login page
> 3. Select a language from the radio button list & activate Submit button
> 4. Find the "Program for Question Lifecycle" card and hit "Apply" in it.
> 5. See the program details on the next page. Hit "Apply" again.
> 6. Fill in the form with dummy data. Activate 'Next' reach page 2 of 3
> 7. Linear navigate and activate ' add entity' button
> 8. backward linear swipe to move onto '' nickname for entity' text field.
> Actual results:upon making the appropriate selection form the list, user has to perform backlward linear swipe gesture to move to Done button. the flow is not linear and user will have to go back to perform selection action which is not a correct flow.
> Expected results:
> Flow should be linear.
